### ISSUE TITLE
Bug: ERROR no iptables supported found

### ISSUE THREAD

**Issue by ChzWheel** [qdm12/gluetun#2264]
### Is this urgent?

No

### Host OS

DSM 7.2.1 Update 5

### CPU arch

x86_64

### VPN service provider

AirVPN

### What are you using to run the container

docker-compose

### What is the version of Gluetun

2024-05-02T08:16:52.183Z (commit e0a977c)

### What's the problem 🤔

iptables error after trying to build docker container.

I saw bug #2260 had been closed with a successful fix, but I seem to still have issues. I attempted running older versions (v3.38) but came across a different error about netip.parseprefix

I know there is not a minimum of 10 lines of logs, but I'm not sure what else I can add since I only get the same 5 lines repeated.

### Share your logs (at least 10 lines)

```plain text
INFO [routing] default route found: interface eth0, gateway 172.20.0.1, assigned IP 172.20.0.7 and family v4
INFO [routing] local ethernet link found: eth0
INFO [routing] local ipnet found: 172.20.0.0/16
ERROR no iptables supported found: errors encountered are: iptables-nft: iptables v1.8.10 (nf_tables): Could not fetch rule set generation id: Invalid argument (exit status 4); iptables: iptables v1.8.10 (nf_tables): Could not fetch rule set generation id: Invalid argument (exit status 4)
INFO Shutdown successful
```

### Share your configuration

```yml
gluetun:
    image: qmcgaw/gluetun:latest
    cap_add:
      - NET_ADMIN
    ports:
      - 8888:8888/tcp
      - 8388:8388/tcp
      - 8388:8388/udp
      - 8090:8090/tcp
      - 9696:9696/tcp
    environment:
      - VPN_SERVICE_PROVIDER=airvpn
      - VPN_TYPE=wireguard
      - WIREGUARD_PRIVATE_KEY=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
      - WIREGUARD_PRESHARED_KEY=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
      - SERVER_COUNTRIES=Netherlands
      - WIREGUARD_ADDRESSES=xx.xxx.xxx.xx
      - FIREWALL_OUTBOUND_SUBNETS=172.20.0.0/16,192.168.0.0/24
    network_mode: synobridge
    restart: always
```

**Comment by Ikzy** [comment]
Hey, I have the same problem.

changing the version from :latest to :v3.3.38 works as a temporary solution.

cheers

**Comment by ChzWheel** [comment]
I had tried switching to v3.38, but ran into an environmental variable error regarding the WIREGAURD_ADDRESSES variable. Since I was getting that issue with the latest version, I didn't troubleshoot much further.

**Comment by frepke** [comment]
Maybe because the typo WIREGAURD.

**Comment by qdm12** [comment]
@ChzWheel @Ikzy `docker pull qmcgaw/gluetun` and it should do it. That bug was introduced in 4d002a3ad68f6c928c6cf8ec683a6f1cfdc3910b and fixed with ef6874fe57500ec322469af4d6781e7e4a9e0719 as well as with fb145d68a00c06cd050a6f1560a392768eb06434 (TLDR: your kernel is quite old, Synology is to blame for lagging behind)

**Comment by ChzWheel** [comment]
It still wasn't working but turns out it was user-error. I was under the impression that using the "latest" tag would automatically pull the latest docker image. But I actually had to go to the list of images where there was an "update available" button. Thanks for the responses.
